### ISSUE TITLE
Rgb0 to Yuv420p example

### ISSUE THREAD

**Issue by felicemarra** [astronautlabs/libyuv-node#6]
If I have a rgba (or rgb0) frame 1280x720 and I want to convert to yuv420p I tried to use RGBAToI420 function but I don't understand what I have to put in the stride parameters. 

` libyuv.RGBAToI420(
                        curFrame.data,
                        1280,
                        planY,
                        1280,
                        planU,
                        1280 / 2,
                        planV,
                        1280 / 2,
                        1280,
                        720
                    );`

![image](https://user-images.githubusercontent.com/2297601/98256547-f7488f00-1f7e-11eb-94c3-30d070768218.png)

this is the result

**Comment by rezonant** [comment]
Yeah adding some examples is a good idea, you have to dig in quite a bit to chromium's source tree to understand.

I'll look into doing that, in the mean time you might find this code from node-webrtc useful-- they have abstracted out the logic that produces the appropriate parameters, but it was very helpful to me to understand what needs to be passed:

https://github.com/node-webrtc/node-webrtc/blob/develop/src/methods/i420_helpers.cc
